CHANGES.rst

```diff
@@ -9,6 +9,7 @@
  - KCWI reduction complete up to spec2D frames
  - Implemented new flatfield DataContainer to separate
    pixelflat and illumflat
+ - Deal with wavecalib crash
```

Comment:
This isn't technically part of 1.0.5 so I made a new category for the hotfixes and added this there.